sips/constant_qbft_timeout.md

```diff
@@ -39,31 +39,58 @@ Attester duties can be submitted with a delay of 31 slots,
 and therefore we can optimize validator performance by having a quick timeout in the first rounds,
 so the committee could have a "second chance" in case the leading operator is faulty.
 
+**Catch up property**   
+Timeouts need to be structured such that nodes in different rounds can "catch" one another and ultimately reach consensus, otherwise a liveness issue is created.
+
 **Quick Timeout Period**
 
 Smaller `T` helps validator performance in the first rounds, 
 but becomes redundant in later rounds when the duty was already finalized.
 
 As a mitigation, we use a quick timeout period where `T=2s`. 
-Once a threshold round was reached the timeout is extended to `T=2m`
+Once a threshold round was reached the timeout becomes larger `T=2m`
+
+**Stop For Catchup**  
+To enjoy both the benefits of a constant timeouts and the catch property we introduce a stop for catchup mechanism. 
+A node will progress through the rounds (via timeout) until it hits round r. At round r it will bump to r+1 but will not start a timeout clock.
```

Comment:
Clarification
```suggestion
A node will progress through the rounds (via timeout) until it hits round r. At round r it will bump to r+1, after receiving f+1 round change messages, but will not start a timeout clock.
```